app/routes/task_routes.py

```diff
@@ -0,0 +1,96 @@
+from app import db
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.helpers.helpers import validate_model, send_message
+from datetime import datetime
+import requests 
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    try:
+        new_task = Task.from_dict(request_body)
+
+        db.session.add(new_task)
+        db.session.commit()
+
+        return {"task" : new_task.to_dict()}, 201
+    
+    except KeyError:
+        abort(make_response({"details": "Invalid data"}, 400))
+
+    db.session.add(new_task)
+    db.session.commit()
+
+
+
+@task_bp.route("", methods=["GET"])
+def get_all_tasks():
+    sort_query = request.args.get("sort")
+    if sort_query:
+        if sort_query == "asc":
+            tasks = Task.query.order_by(Task.title.asc()).all()
+        elif sort_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc()).all()
+    else:
+        tasks = Task.query.all()
+    if not tasks:
+        return make_response(jsonify([]), 200)
```

Comment:
We don't need to explicitly return an empty list. You can remove lines 39-40 and let the rest of the code execute.  Like mentioned in a couple other comments, if tasks is an empty list then looping over it isn't an issue.